core/tests/behavior/blocking_list.rs

```diff
@@ -26,10 +26,17 @@ use crate::*;
 pub fn behavior_blocking_list_tests(op: &Operator) -> Vec<Trial> {
     let cap = op.info().capability();
 
-    if !(cap.read && cap.write && cap.copy && cap.blocking && cap.list) {
+    if !(cap.read && cap.write && cap.copy && cap.list) {
         return vec![];
     }
 
+    let mut op = op.clone();
```

Comment:
We should avoid modifying the blocking test suites. Instead, we can make changes at https://github.com/apache/incubator-opendal/blob/main/core/tests/behavior/utils.rs#L46.

This approach is more aligned with how OpenDAL users will interact with the software.